### ISSUE TITLE
Problems starting container

### ISSUE THREAD

**Issue by JOscarJ** [dtulibrary/freecite-docker#1]
Hello there,
I am finding problems starting docker container
Error is rather misleading:

`Switch to inspect mode.`

I have tried with another ruby versions and the result is the same
```
2017-03-09T17:11:49.194455694+01:00 container create 48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16 (image=freecite:latest, name=adoring_mclean)
2017-03-09T17:11:49.197716456+01:00 container attach 48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16 (image=freecite:latest, name=adoring_mclean)
2017-03-09T17:11:49.224141395+01:00 network connect 5c161475473ed523b1c1aaf63aa6c425594cac2a9ea45fd1aab5d540a06c56f6 (container=48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16, name=bridge, type=bridge)
2017-03-09T17:11:49.418633011+01:00 container start 48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16 (image=freecite:latest, name=adoring_mclean)
Switch to inspect mode.

2017-03-09T17:11:49.609411357+01:00 container die 48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16 (exitCode=0, image=freecite:latest, name=adoring_mclean)
2017-03-09T17:11:49.758363787+01:00 network disconnect 5c161475473ed523b1c1aaf63aa6c425594cac2a9ea45fd1aab5d540a06c56f6 (container=48f492179e785c4995ba9163532be943d51cf58e46ec154b157a024e9fefad16, name=bridge, type=bridge)
```

Environment
Ubuntu 15.10
Docker version 1.12.6, build 78d1802

**Comment by ronan-mch** [comment]
Hi Oscar,

We're off work for the evening, I'll ask @troelslisberg if he can help you when I see him tomorrow.

**Comment by troelslisberg** [comment]
The Dockerfile did not contain a CMD line since this project is only used as part of a docker-compose project (which then specifies the CMD). I have added a CMD line to the Docker file.

Please try again @JOscarJ .

**Comment by JOscarJ** [comment]
Thank you for the quick response

Now container is working and listening but URL is always responding Not Found: /
We have tried every port possible 0.0.0.0:9292, 0.0.0.0:3000, localhost:9292, localhost:3000, localhost:3000/citations/create but Not Found: /

In ps -aux is running a process

`ruby /usr/local/bundle/bin/rackup --host 0.0.0.0`

**Comment by troelslisberg** [comment]
Try this url: http://localhost:9292/citation/search?q=Huber%2C%20A.%20K.%3B%20Falk%2C%20M.%3B%20Rohnke%2C%20M.%3B%20Luerssen%2C%20B.%3B%20Gregoratti%2C%20L.%3B%20Amati%2C%20M.%3B%20Janek%2C%20J.%20Physical%20Chemistry%20Chemical%20Physics%202012%2C%2014%2C%20751

**Comment by JOscarJ** [comment]
Hello troelslisberg,
this command works indeed, but controller only implements method /search of the API
is it possible to have /create and POST requests in order to let it insert cites in its database?

**Comment by myedibleenso** [comment]
I'm also needing to use `/create`.  Is adding support for method planned?

**Comment by ronan-mch** [comment]
I don't think so, DTU just forked this project to test it. I don't think that it's in active development.
